src/storage/config.rs

```diff
@@ -135,6 +143,11 @@ impl ConfigManager for StorageConfigManger {
                         .set(size.0 as f64);
                 }
             }
+        } else if let Some(v) = change.remove("ttl_check_poll_interval") {
```

Comment:
Any test case?

Reply:
See test case in src/config.rs

Reply:
covered by `test_change_ttl_check_poll_interval`, and I tested it manually too
![image](https://user-images.githubusercontent.com/13497871/111098540-a83ee600-857e-11eb-9880-5208d0b37219.png)